src/polygon2d.jl

```diff
@@ -0,0 +1,43 @@
+"""
+```
+draw_polygon_perimeter!(img, vertices)
+img2=draw_polygon_perimeter(img, vertices)
+```
+Draws the perimeter of the polygon formed by input vertices(Array of CartesianIndex).
+"""
+
+function draw_polygon_perimeter{T<:Colorant}(img::AbstractArray{T, 2}, vertices::Array{CartesianIndex{2},1})
```

Comment:
You could use the `draw_polygon_perimeter!` function by passing `copy(img)` as the first argument to avoid duplicating code. You could check out how the `line` function is implemented.